OpenRA.Mods.TS/UtilityCommands/ImportIsometricLegacyMapCommand.cs

```diff
@@ -0,0 +1,476 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2014 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using OpenRA.Mods.Common.Traits;
+using OpenRA.FileFormats;
+using OpenRA.FileSystem;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.TS.UtilityCommands
+{
+	public class Format5
```

Comment:
This class does not follow our coding conventions, and probably doesn't want to live here.